tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionEpicRepositoryTests.cs

```diff
@@ -0,0 +1,180 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NHSD.BuyingCatalogue.Solutions.Contracts.Persistence;
+using NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests.Models;
+using NHSD.BuyingCatalogue.Testing.Data;
+using NHSD.BuyingCatalogue.Testing.Data.Entities;
+using NHSD.BuyingCatalogue.Testing.Data.EntityBuilders;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests
+{
+    [TestFixture]
+    public sealed class SolutionEpicRepositoryTests
+    {
+        private const string Solution1Id = "Sln1";
+
+        private const string SupplierId = "Sup 1";
+
+        private const string StatusPassed = "Passed";
+
+        private ISolutionEpicRepository _solutionEpicRepository;
+
+        private static readonly List<CapabilityDetails> _capDetails = new List<CapabilityDetails>()
+        {
+            CreateCapability("Cap1", "Desc1", "Ref1", "1.0.0","http://cap1.link"),
+            CreateCapability("Cap2", "Desc2", "Ref2","1.0.0","http://cap2.link")
+        };
+
+        private readonly List<EpicDetails> _epicDetails = new List<EpicDetails>()
+        {
+            CreateEpic("Epic1", "Name1", _capDetails[0].Id),
+            CreateEpic("Epic2", "Name2", _capDetails[1].Id)
+        };
+
+        private static CapabilityDetails CreateCapability(string name, string desc, string reference, string version, string sourceUrl)
```

Comment:
Could we introduce an CapabilityDetailsBuilder similar to the other builders but just for test object creation?